tools/dev/loki-boltdb-storage-s3/config/loki.yaml

```diff
@@ -1,4 +1,4 @@
-auth_enabled: false
+auth_enabled: true 
```

Comment:
I can change it back. However, I don't think it this example wasn't used much as it wasn't working due to changes in the Minio image.